data/Localization/german_(germany)/global.ini

```diff
@@ -64623,7 +64623,7 @@ redwind_SignOff_001=\Nehmen Sie diesen Vertrag nicht an - es sei denn, Sie sind
 redwind_SignOff_002=\n\nBE AWARE - Rot Wind ist nicht nur als zuverlässiger Versanddienstleister bekannt, sondern auch dafür, Leute aufzuspüren, die uns hintergehen. Wenn Sie auf der Suche nach einer leichten Beute sind, sind wir nicht die Richtigen. Nur vertrauenswürdige Piloten.
 redwind_SignOff_003=AN ALLE VETS - Rot Wind schätzt euren Dienst für unser Reich! Eure Kämpfe bedeuten unsere Freiheit!
 redwind_SignOff_004=WIR SIND EQUAL OPPORTUNITY - Rot Wind Linehaul ist und war schon immer ein Unternehmen, das auf Chancengleichheit setzt. Wenn Sie glauben, dass Sie einen guten Job für uns machen können, dann melden Sie sich auf jeden Fall.
-redwind_SignOff_005=VERTRAUEN GEHT IN ZWEI WEGEN - Mit der Annahme dieses Vertrags verpflichten Sie sich, uns nicht zu bescheißen, und Rot Wind Linehaul verpflichtet sich, Sie genauso zu behandeln.
+redwind_SignOff_005=VERTRAUEN GEHT IN ZWEI WEGEN - Mit der Annahme dieses Auftrags verpflichten Sie sich, uns nicht zu bescheißen, und Rot Wind Linehaul verpflichtet sich, Sie genauso zu behandeln.
```

Comment:
Schlage vor das zu ändern in : ""GEGENSEITIGES VERTRAUEN IST ESSENTIELL - Indem Sie diesen Vertrag annehmen, verpflichten Sie sich, uns nicht zu hintergehen, und Red Wind Linehaul verspricht, Sie genauso zu behandeln.""